app/config/config.yml.dist

```diff
@@ -204,8 +204,8 @@ thumbnails:
     default_image: [ 1000, 750 ]
     quality: 80
     cropping: crop
-    notfound_image: view://img/default_notfound.png
-    error_image: view://img/default_error.png
+    notfound_image: bolt://app/view/img/default_notfound.png
+    error_image: bolt://app/view/img/default_error.png
```

Comment:
Just flagging this for others, as per @CarsonF's  comment on the PR itself.

My concern with "public facing" use of `bolt://` is that to a large part of our target audience of implementers, it might appear that `bolt://` is the prefix for all Bolt files, rather than the root of the code.

I personally don't have a merge-blocking concern, but did want to flag it for those advocating for other target user groups … ping @bobdenotter